### PR TITLE
More syscall coverage improvements

### PR DESCRIPTION
**Pull request by Keno**
All of these system calls are pretty boring from our perspective, so the bulk of this is tests.

### CODE REVIEW

File: src/test/pivot_root.c [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/* -*- Mode: C; tab-width: 8; c-basic-offset: 2; indent-tabs-mode: nil; -*- */
+
+#include "nsutils.h"
+#include "rrutil.h"
+
+int main(void) {
+  try_setup_ns(CLONE_NEWNS);
```
Comment (rocallahan) [inline src/test/pivot_root.c]: Shouldn't we print EXIT-SUCCESS here and exit if this fails?

Alternatively, `try_setup_ns` could just do that itself after it prints the `Skipping test` warning.

File: src/test/privileged_net_ioctl.c [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+  // sudo brctl delbr rrbridge0
+  {
+    char buf[IFNAMSIZ];
+    strncpy(buf, bridge_name, sizeof(buf));
+    test_assert(0 == ioctl(inetfd, SIOCBRDELBR, buf));
+  }
+
+  atomic_printf("EXIT-SUCCESS");
+  _exit(0);
```
Comment (rocallahan) [inline src/test/privileged_net_ioctl.c]: just `return 0;`?

File: src/test/pivot_root.c (hunk 1 above)
Comment (Keno) [inline src/test/pivot_root.c]: Yes, it should thanks. I considered having this be inside the implementation, but then it seemed better for it to not be quite that magic and have the skipping be quite explicit.

**Comment by Keno** [comment]
Comments addressed.

**Comment by rocallahan** [comment]
Excellent, thanks!

**Comment by Keno** [comment]
My pleasure. Maybe someday soon I'll be able to pick up an executable without having to find myself adding syscalls to rr ;).

**Pull request merged**